test/chrome_test.js

```diff
@@ -141,4 +141,55 @@ describe("StorageArea", function() {
     });
   });
 
+  /** @test {StorageArea#remove} */
+  describe("#remove", function() {
+    it("removes a single entry", function(done) {
+      area.set({ foo: "bar" }, function() {
+        area.remove("foo", function() {
+          area.get("foo", function(items) {
+            expect(items).to.deep.eql([]);
+            done();
+          });
+        });
+      });
+    });
+
+    it("silently ignores a missing entry", function(done) {
+      area.remove("nonexisting", function() {
+        done();
+      });
+    });
+
+    it("does not remove other entries", function(done) {
+      area.set({ foo: "bar", baz: "bar" }, function() {
+        area.remove("baz", function() {
+          area.get("foo", function(items) {
+            expect(items.foo).to.eql("bar");
+            done();
+          });
+        });
+      });
+    });
+
+    it("removes an array of keys", function(done) {
+      area.set({ foo: "bar", baz: "bar" }, function() {
+        area.remove(["foo", "baz"], function() {
+          area.get(["foo", "baz"], function(items) {
+            expect(items).to.deep.eql([]);
+            done();
+          });
+        });
+      });
+    });
+
+    it("treats the callback parameter as optional", function(done) {
+      area.set({ foo: "bar" }, function() {
+        area.remove(["foo"]);
+        setTimeout(function() {
```

Comment:
As I commented on IRC, we don't really want to bother executing an actual IDB transaction for this unit test, we mostly want to ensure Collection#delete is properly called with the expected data. We could stub Collection#delete for doing just that. WDYT?